dist/src/main/assembly/conf/brooklyn/default.catalog.bom

```diff
@@ -30,11 +30,11 @@ brooklyn.catalog:
         docker.policy.ha.enable: false
         docker.container.strategies:
         - $brooklyn:object:
-            type: clocker.docker.location.strategy.MaxContainersPlacementStrategy
+            type: clocker.docker.location.strategy.basic.MaxContainersPlacementStrategy
```

Comment:
Allowing each jar or project to provide a file with a well-known or discoverable name for a `DeserializingClassRenamesProvider` would be good. I can implement one for the (large amount of) Clocker renaming that was done for the OSGi work, but in another PR.